changefeedtrigger/__init__.py

```diff
@@ -0,0 +1,94 @@
+import time
+import json
+from subprocess import Popen, PIPE
+from azure.storage.blob.changefeed import ChangeFeedClient
+from azure.storage.blob import BlobServiceClient
+import azure.functions as func
+from datetime import timedelta, datetime
+import logging
+from predicate import where
+
+
+def multidimensional_filtering(cf_client, b_filter, blob_client, cont_name, c_filter, e_filter):
+    filter_event_service = blob_client.get_container_client(container=cont_name)
+    blob_name = "continuation-token"
+
+    # access blobs in container
+    blob_list = filter_event_service.list_blobs()
+    tokens = []
+    for blob in blob_list:
+        tokens.append(blob.name)
+    
+    # check if continuation token already exists
+    if blob_name in tokens:
```

Comment:
This looks like quite complicated way to check if blob exists. In other languages we have `BlobClient.Exists` APIs - I'm quite surprised to not see that in python. @xiafu-msft  is there any better way to check if blob exists ?

Reply:
I have not been able to find anything like the .exists() that is in java (and maybe .net). But @xiafu-msft please let me know if you know of a more simple way!